superchain/configs/mainnet/lisk.toml

```
@@ -6,6 +6,7 @@ explorer = "https://blockscout.lisk.com"
 superchain_level = 0
 standard_chain_candidate = true # This is a temporary field which causes most of the standard validation checks to run on this chain
 batch_inbox_addr = "0xFf00000000000000000000000000000000001135"
+superchain_time = 0
 canyon_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
 delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
 ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
```

```suggestion
superchain_time = 0
```
and you can also delete the two entries for `fjord_time` and `granite_time` because they are the same as the superchain times. Github didn't let me include these two lines in the diff proposal.